portfolio/src/main/webapp/script.js

```diff
@@ -125,3 +126,83 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+/**
+ * Function that inits the Google Maps map
+ */
+function initMap() {
+  // Set locations coordinates
+  const homeAlexandriaCoords = { 
+    lat: 43.975320, 
+    lng: 25.316876 
+  };
```

Comment:
I guess in google codestyle they also put a trailing comma. [link](https://google.github.io/styleguide/jsguide.html#features-object-literals)

Reply:
Done, thanks for the observation!